packages/shared/src/graphql/squads.ts

```diff
@@ -167,3 +217,24 @@ export async function getSquadMembers(id: string): Promise<SquadMember[]> {
   );
   return res.sourceMembers.edges?.map((edge) => edge.node);
 }
+
+interface SquadInvitation {
+  member: SquadMember;
+}
+
+export interface SquadInvitationProps {
+  token: string;
+  sourceId: string;
+}
+
+export const getSquadInvitation = (token: string): Promise<SquadInvitation> =>
+  request(`${apiUrl}/graphql`, SQUAD_INVITATION_QUERY, { token });
+
+export const joinSquadInvitation = ({
+  token,
+  sourceId,
+}: SquadInvitationProps): Promise<SquadInvitation> =>
+  request(`${apiUrl}/graphql`, SQUAD_JOIN_MUTATION, {
+    token,
+    sourceId: sourceId.toLowerCase(),
```

Comment:
This is always lowercase already, right?

Reply:
Ah, yes, this is source id which is UUID, so no need to lowercase.